lib/hiera/backend.rb

```diff
@@ -307,8 +307,19 @@ def clear!
 
       def qualified_lookup(segments, hash)
         value = hash
+
         segments.each do |segment|
           throw :no_such_key if value.nil?
+          if segment =~ /^::(.+)/
```

Comment:
Now that I think about it, we should be able to fix this from `bin/hiera`. Going to try just adding entries to options[:scope] rather than overriding default_proc.